pubsub/gossipsub/gossipsub-v1.1.md

```diff
@@ -0,0 +1,374 @@
+# gossipsub v1.1: gossipsub extensions to improve bootstrapping and attack resistance
+
+| Lifecycle Stage | Maturity       | Status | Latest Revision |
+|-----------------|----------------|--------|-----------------|
+| 1A              | Draft          | Active | r1, 2020-03-20  |
+
+
+Authors: [@vyzo]
+
+Interest Group: [@yusefnapora], [@raulk], [@whyrusleeping], [@Stebalien], [@daviddias]
+
+[@whyrusleeping]: https://github.com/whyrusleeping
+[@yusefnapora]: https://github.com/yusefnapora
+[@raulk]: https://github.com/raulk
+[@vyzo]: https://github.com/vyzo
+[@Stebalien]: https://github.com/Stebalien
+[@daviddias]: https://github.com/daviddias
+
+See the [lifecycle document][lifecycle-spec] for context about maturity level
+and spec status.
+
+[lifecycle-spec]: https://github.com/libp2p/specs/blob/master/00-framework-01-spec-lifecycle.md
+
+---
+
+## Overview
+
+This document specifies extensions to [gossipsub v1.0](gossipsub-v1.0.md) intended to improve
+bootstrapping and protocol attack resistant. The extensions change the algorithms that
+prescribe local peer behaviour and are fully backwards compatible with v1.0 of the protocol.
+Peers that implement these extensions, advertise v1.1 of the protocol using `/meshsub/1.1.0`
+as the protocol string.
+
```

Comment:
```suggestion

# Considered Attack Vectors

`To be written`

```

It would be good to add a section that we briefly explain the Attack Vectors we considered as it will inform the reader how the extensions proposed impact them.